gadgets/trace_exec/README.mdx

```diff
@@ -41,12 +41,6 @@ Show current working directory, file path and executable path. The file path and
 
 Default value: "false"
 
-### `--uid`
-
-Show only events generated by processes with this uid
-
-Default value: ""
```

Comment:
It seems I was mistaken, and there was no need to run the Makefile to update README.mdx. Sorry about that.

I thought dev_major, dev_minor and inode would automatically get added in the documentation. But the Makefile automatically adds documentation about parameters and not fields.

I am not sure why the Makefile removed `--uid` (maybe related to #3648) but in any case, it is not related to your PR, so please add it back.